### ISSUE TITLE
Issue with globby "cwd" not getting a string 

### ISSUE THREAD

**Issue by FireInjun** [jorgegonzalez/zoe#84]
```
Zoe 1.24.2
TypeError: Expected `cwd` to be of type `string` but received type `undefined`
    at module.exports.sync (N:\fireinjun.github.io\reactRewrite\mainjs\node_modules\dir-glob\index.js:60:9)
    at globDirs (N:\fireinjun.github.io\reactRewrite\mainjs\node_modules\globby\index.js:58:9)
    at getPattern (N:\fireinjun.github.io\reactRewrite\mainjs\node_modules\globby\index.js:61:64)
    at globTasks.reduce (N:\fireinjun.github.io\reactRewrite\mainjs\node_modules\globby\index.js:107:19)
    at Array.reduce (<anonymous>)
    at Function.module.exports.sync (N:\fireinjun.github.io\reactRewrite\mainjs\node_modules\globby\index.js:106:26)
    at findPaths (N:\fireinjun.github.io\reactRewrite\mainjs\node_modules\zoe\dist\engine.js:27:29)
    at Object.engine [as default] (N:\fireinjun.github.io\reactRewrite\mainjs\node_modules\zoe\dist\engine.js:36:24)
    at N:\fireinjun.github.io\reactRewrite\mainjs\node_modules\zoe\dist\zoe.js:36:41
    at Generator.next (<anonymous>)
```
I don't know if it's platform independent as I don't use bash so I double checked with gitbash and mintty. So then I tried manually inserting a glob and running everything via `npx`.

**Comment by jorgegonzalez** [comment]
I haven't had much time to look into this myself, but I will absolutely accept a PR fixing this issue.

**Comment by FireInjun** [comment]
If I get a chance, I'll start digging into it for you

**Comment by soulchainer** [comment]
@FireInjun @jorgegonzalez After some quick digging, considering the date this was reported and the version of zoe (1.24.2), checking the [`package.json` back  then](https://github.com/jorgegonzalez/zoe/blob/20eb85f691248520556e1b3561503064690f0935/zoe/package.json#L73) and the version of `globby` that was in use, I bet this was related with [this globby issue, version `8.0.2`](https://github.com/sindresorhus/globby/issues/106), which was solved in version `8.0.3` (and was all about version requirements in `package.json`).

We are already using [`"globby": "10.0.1"`](https://github.com/jorgegonzalez/zoe/blob/master/package.json#L60), so far ahead from that.

Considering this, the time that has passed and the fact that there is more info needed to even reproduce the issue, I think we could close it for now, right? It can be reopened anytime if the error resurfaces.
